hadoop-ozone/dist/src/main/k8s/examples/testlib.sh

```diff
@@ -77,6 +77,13 @@ start_k8s_env() {
    wait_for_startup
 }
 
+get_logs() {
+  mkdir -p logs
+  for pod in $(kubectl get pods -o custom-columns=NAME:.metadata.name | tail -n +2); do
```

Comment:
It's a shame that we have no better solution. We can do `kubectl logs -l app=ozone` but it will fail when we add setup with more apps (like ozone + spark).

FTR: locally I use https://github.com/wercker/stern and it works well (but this loop is easier.)